spec/features/peep_management_spec.rb

```diff
@@ -0,0 +1,140 @@
+feature 'Peeps shown on website' do
+
+  before(:each) do
+    sign_up
+    sign_in
+    Peep.create(peep_text: 'Hello there!')
+  end
+
+  def sign_up(name="Ashleigh Smells",
+              username="ashleighpants",
+              email="ash@ash.com",
+              password="pass",
+              password_confirmation="pass")
+    visit '/users/new'
+    expect(page.status_code).to eq 200
+    fill_in :name, with: name
+    fill_in :username, with: username
+    fill_in :email, with: email
+    fill_in :password, with: password
+    fill_in :password_confirmation, with: password_confirmation
+    click_button 'Sign Up'
+  end
```

Comment:
Nice! This makes tests extra-legible. For many extra awards, put this in a `private` area down the bottom of this Example Group.